lang/main-pl.json

```diff
@@ -824,31 +849,32 @@
         },
         "calendar": "Kalendarz",
         "connectCalendarButton": "Podłącz swój kalendarz",
-        "connectCalendarText": "",
+        "connectCalendarText": "Connect your calendar to view all your meetings in {{app}}. Plus, add {{provider}} meetings to your calendar and start them with one click.",
```

Comment:
Why are these in English? You better remove those, it will anyway fallback to English or translate them. There is at least one more like that above ^

Having empty string or English one once we switch to some translate service ... This will confuse it

Reply:
I didn't notice them

It should be translated properly now.